homeassistant/components/vizio/__init__.py

```diff
@@ -1 +1,43 @@
 """The vizio component."""
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    CONF_DEVICE_CLASS,
+    CONF_HOST,
+    CONF_NAME,
+)
+from homeassistant.helpers import config_validation as cv
+
+from .const import (
+    CONF_SUPPRESS_WARNING,
+    CONF_VOLUME_STEP,
+    DEFAULT_DEVICE_CLASS,
+    DEFAULT_NAME,
+    DEFAULT_VOLUME_STEP,
+)
+
+
+def validate_auth(config):
+    """Validate presence of CONF_ACCESS_TOKEN when CONF_DEVICE_CLASS=tv."""
+    token = config.get(CONF_ACCESS_TOKEN)
+    if config[CONF_DEVICE_CLASS] == "tv" and (token is None or token == ""):
```

Comment:
```py
if config[CONF_DEVICE_CLASS] == "tv" and not token:
```

Reply:
🤦‍♂ I had already made this change but forgot to add it to the commit. Thanks!